pinot-core/src/main/java/org/apache/pinot/core/operator/InstanceResponseOperator.java

```diff
@@ -34,24 +34,21 @@
 
 
 public class InstanceResponseOperator extends BaseOperator<InstanceResponseBlock> {
-
   private static final String EXPLAIN_NAME = "INSTANCE_RESPONSE";
 
-  private final BaseCombineOperator _combineOperator;
+  private final BaseCombineOperator<?> _combineOperator;
```

Comment:
should we also put type generic here for InstanceResponseOperator<T extends BaseResultBlock> ? 
eventually resultsBlock.getDataTable(QueryContext) is implemented at each individual `T` correct?

Reply:
We only need that in the combine operator because we need to know the `T` when merging the blocks. the `getDataTable()` is a common interface, so we don't need to know the exact type